PathGenerator/PathGeneratorModel.cpp

```diff
@@ -0,0 +1,84 @@
+#include "PathGenerator.hpp"
+#include <iostream>
+#include <cmath>
+
+namespace Example
+{
+void PathGenerator::operator()(const halp::tick_flicks& t)
+{
+  // Linear path
+  if (inputs.Path == 0) {
+    std::cout << (int)t.relative_position << std::endl;
```

Comment:
don't leave std::cout in the final commits as they can really break the real-time performance since this is all running in the audio thread